expander.go

```diff
@@ -231,6 +245,37 @@ func nextRef(startingNode interface{}, startingRef *Ref, ptr *jsonpointer.Pointe
 	return ret
 }
 
+func normalizeFileRef(ref *Ref, relativeBase string) *Ref {
+	refURL := ref.GetURL()
+
+	if !strings.HasPrefix(refURL.String(), "#") {
```

Comment:
perhaps it's a bit cleaner if you return when it has the prefix, reduces 1 level of nesting